internal/chart/application_chart.go

```diff
@@ -329,33 +333,39 @@ func isAppAccessible(a *app) bool {
 
 func newIngress(app ketchv1.App, framework ketchv1.Framework) (*ingress, error) {
 	var http []string
-	var https []string
+	var https []httpsEndpoint
 
 	for _, cname := range app.Spec.Ingress.Cnames {
 		if cname.Secure {
-			if len(framework.Spec.IngressController.ClusterIssuer) == 0 {
-				return nil, errors.New("secure cnames require a framework.Ingress.ClusterIssuer to be specified")
+			secretName := app.Spec.SecretName
+			clusterIssuer := fmt.Sprintf("%s-clusterissuer", app.Spec.SecretName)
```

Comment:
do we need to initialize them inside this loop for every iteration? Can they be outside the loop?

Reply:
I cleaned this up as part of enabling multiple secrets.